engine/api/worker/model.go

```diff
@@ -426,6 +426,10 @@ func updateAllToCheckRegistration(db gorp.SqlExecutor) error {
 
 // UpdateSpawnErrorWorkerModel updates worker model error registration
 func UpdateSpawnErrorWorkerModel(db gorp.SqlExecutor, modelID int64, spawnError sdk.SpawnErrorForm) error {
+	if spawnError.Error == string([]byte{0x00}) {
```

Comment:
When is this happening ? Can you add a comment ?

Reply:
some times when the docker container fails to start, the docker logs is not empty but only contains utf8 null char

Reply:
comment added